learning/index.md

```diff
@@ -30,6 +30,7 @@ The [Julia Language's YouTube channel](https://www.youtube.com/user/JuliaLanguag
 ## Text
 
 @@tight-list
+* [From zero to Julia!](https://techytok.com/from-zero-to-julia/) by [Aurelio Amerio](https://github.com/aurelio-amerio). An expanding series of short lessons about Julia, starting from beginner level and going up to deal with more advanced topics.
```

Comment:
```suggestion
* [From zero to Julia!](https://techytok.com/from-zero-to-julia/) by [Aurelio Amerio](https://github.com/aurelio-amerio). An expanding series of short lessons about Julia, starting from the beginner level and going up to deal with the more advanced topics.
```
I think the current description is very good @tlienart. For some reason `from introductory to more advanced topics` sounds more like a description for a book on theoretical mathematics. :smile:
But maybe `tutorials` does sound better than `lessons`.

Reply:
👍

Reply:
@tlienart I like your conciseness and I've swapped lessons with tutorials. Though I wanted to put an emphasis to the fact that it is suitable for people who have little to no knowledge of programming... 
@zdroid I've added the "the"